sdk/sdk.go

```diff
@@ -306,23 +258,13 @@ func buildIDPTokenSource(c *config) (auth.AccessTokenSource, error) {
 	return ts, err
 }
 
-// Close closes the underlying grpc.ClientConn.
+// TODO: Remove after otdfctl updates
 func (s SDK) Close() error {
```

Comment:
can remove this after otdfctl changes, dont want to block xtests

Reply:
Don't we need to clean up the collection store?